bdd/features/pilot/pilot_transaction.feature

```diff
@@ -36,6 +61,17 @@ Feature: A transaction is generated, authorized and confirmed
     Then the transaction X is authorized
     And the citizen A is rewarded with 300 euros
 
+  @transaction
+  @Scontoditipo1
+  @MIL
+  Scenario: The budget is completely eroded with one trx through MIL
+    Given the merchant 1 is qualified
+    And the citizen A is onboard
+    And the merchant 1 generates the transaction X of amount 30000 cents through MIL
+    When the citizen A confirms the transaction X
+    Then the transaction X is authorized
+    And the citizen A is rewarded accordingly
```

Comment:
@Benedetta-fabbri every "the citizen A is rewarded accordingly" should specify the exact expected reward 😊 (in this case "the citizen A is rewarded with 300 euros")

Reply:
Yes, fixed it